doc/api/api_changes_3.4/deprecations.rst

```diff
@@ -1,6 +1,20 @@
 Deprecations
 ------------
 
+Figure.colorbar now warns when the mappable's axes is different from the current axes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Currently, `.Figure.colorbar` steals space by default from the current axes
+to place the colorbar.  In a future version, it will steal space from the
+mappable's axes instead.  In preparation for this change, `.Figure.colorbar`
+now emits a warning when the current axes is not the same as the mappable's
+axes.
+
+The ``.ax`` attribute to `.ContourSet` is deprecated in favor of
+``.axes`` for consistency with artists (even though `.ContourSet` is not an
+`.Artist` subclass).
```

Comment:
Can be removed because it's been extracted to #16058.

Reply:
yup